src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/postsubmit/SendDirectionConfirmation.java

```diff
@@ -27,20 +28,35 @@ public PostSubmitCallbackResponse handle(
             throw new IllegalStateException("Cannot handle callback");
         }
 
+        final AsylumCase asylumCase = callback.getCaseDetails().getCaseData();
+        final String hoAmendBundleReadyInstructStatus =
+            asylumCase.read(AsylumCaseFieldDefinition.HOME_OFFICE_AMEND_BUNDLE_INSTRUCT_STATUS,
```

Comment:
I think this will start throwing error without corrresponding ccd-definitions change. Have tested this by creating a case and checking the confirmation. Could you please check it once?

Reply:
ccd-def changed.
Also changed handler to call ho api only for state "awaitingRespondentEvidence"